lib/elements/element.py

```diff
@@ -135,6 +136,32 @@ def get_int_param(self, param, default=None):
 
         return value
 
+    # returns 2 float values as a numpy array
+    # if a single number is given in the param, it will apply to both returned values.
+    # Not cached the cache will crash if the default is a numpy array.
+    # The ppoperty calling this will need to cache itself and can safely do so since it has no parameters
+    def get_lr_float_param(self, param, default=(0, 0)):
+        default = np.array(default)  # type coersion in case the default is a tuple
+
+        raw = self.get_param(param, "")
+        parts = raw.split()
+        try:
+            if len(parts) == 0:
+                return default
+            elif len(parts) == 1:
+                a = float(parts[0])
+                return np.array([a, a])
+            else:
+                a = float(parts[0])
+                b = float(parts[1])
+                return np.array([a, b])
+        except (TypeError, ValueError):
+            return default
+
+    # not cached
+    def get_lr_mm_param_as_px(self, param, default):
```

Comment:
If a float or int param ends on _mm it will automatically calculate the px when you use get_float_param or get_int_param

Reply:
Ah, sorry. This happened, because you have two values to process.

Reply:
Also, I want to split `get_float_param` into `get_float_param` and `get_mm_param_as_px` in a future PR as I found the current behavior quite confusing. I did two versions of the 2-valued version so is not to propagate the magic.